app/models/board.rb

```diff
@@ -16,6 +16,10 @@ def to_param
     slug
   end
 
+  def previous_boards
```

Comment:
nope, this is bad, especially the way you use it. Please use service at the place you need it, not in model